lib/aerospike/node/refresh/peers.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Aerospike
+  class Node
+    module Refresh
+      module Peers
+        class << self
+          def call(node, peers)
+            return unless should_refresh?(node)
+
+            ::Aerospike.logger.debug("Update peers for node #{node.name}")
+
+            cluster = node.cluster
+
+            collection = ::Aerospike::Peers::Fetch.(cluster, node.tend_connection)
+            peers.peers = collection.peers
+            node.peers_count.value = peers.peers.size
+            peers_validated = true
+
+            peers.peers.each do |peer|
+              next if ::Aerospike::Cluster::FindNode.(cluster, peers, peer.node_name)
+
+              node_validated = false
```

Comment:
Should this be `peers_validated`? (And on lines 34 and 41 as well.)

Reply:
Turned out I missed this part of transcribing the Java peer refresh code
https://github.com/aerospike/aerospike-client-java/blob/7352c15c04015f4bd4f6985e138fa931179fb5d7/client/src/com/aerospike/client/cluster/Node.java#L363-L365